torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -734,6 +734,18 @@ void LLVMCodeGenImpl::visit(const Ramp* v) {
   auto stride = this->value_;
   int lanes = v->lanes();
 
+  if (llvm::ConstantInt* const_stride = llvm::dyn_cast<llvm::ConstantInt>(stride)) {
+    std::vector<llvm::Constant*> vals = { llvm::ConstantInt::get(base->getType(), 0) };
+    for (int i = 1; i < lanes; ++i) {
```

Comment:
Nit: is `lanes` signed or unsigned? If it's unsigned, then clang-tidy will complain either about this or 735 line.

Reply:
It's signed